docs/user/azure/install_upi_azurestack.md

```diff
@@ -0,0 +1,606 @@
+next steps we're going to deploy each one of them in order, using [az (Azure CLI)][azurecli] and providing the expected parameters.
+
+## Deploy the Virtual Network
+
+In this example we're going to create a Virtual Network and subnets specifically for the OpenShift cluster. You can skip this step
+if the cluster is going to live in a VNet already existing in your organization, or you can edit the `01_vnet.json` file to your
+own needs (e.g. change the subnets address prefixes in CIDR format).
+
+Copy the [`01_vnet.json`](../../../upi/azurestack/01_vnet.json) ARM template locally.
+
+Create the deployment using the `az` client:
+
+```sh
+az deployment group create -g $RESOURCE_GROUP \
+  --template-file "01_vnet.json" \
+  --parameters baseName="$INFRA_ID"
+```
+
+## Deploy the image
+
+Copy the [`02_storage.json`](../../../upi/azurestack/02_storage.json) ARM template locally.
+
+Create the deployment using the `az` client:
+
+```sh
+export VHD_BLOB_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c vhd -n "rhcos.vhd" -o tsv`
+
+az deployment group create -g $RESOURCE_GROUP \
+  --template-file "02_storage.json" \
+  --parameters vhdBlobURL="$VHD_BLOB_URL" \
+  --parameters baseName="$INFRA_ID"
+```
+
+## Deploy the load balancers
+
+Copy the [`03_infra.json`](../../../upi/azurestack/03_infra.json) ARM template locally.
+
+Deploy the load balancer and public IP addresses using the `az` client:
+
+```sh
+az deployment group create -g $RESOURCE_GROUP \
+  --template-file "03_infra.json" \
+  --parameters baseName="$INFRA_ID"
+```
+
+Create an `api` and `api-int` DNS record in the *public* zone for the API public load balancer. Note that the `BASE_DOMAIN_RESOURCE_GROUP` must point to the resource group where the public DNS zone exists.
+
+```sh
+export PUBLIC_IP=$(az network public-ip list -g "$RESOURCE_GROUP" --query "[?name=='${INFRA_ID}-master-pip'] | [0].ipAddress" -o tsv)
+az network dns record-set a add-record -g "$RESOURCE_GROUP" -z "${CLUSTER_NAME}.${BASE_DOMAIN}" -n api -a "$PUBLIC_IP" --ttl 60
+az network dns record-set a add-record -g "$RESOURCE_GROUP" -z "${CLUSTER_NAME}.${BASE_DOMAIN}" -n api-int -a "$PUBLIC_IP" --ttl 60
+```
+
+## Launch the temporary cluster bootstrap
+
+Copy the [`04_bootstrap.json`](../../../upi/azurestack/04_bootstrap.json) ARM template locally.
+
+Create the deployment using the `az` client:
+
+```sh
+export BOOTSTRAP_URL=$(az storage blob url --account-name "${INFRA_ID}sa" --account-key "$ACCOUNT_KEY" -c "files" -n "bootstrap.ign" -o tsv)
+export BOOTSTRAP_IGNITION=$(jq -rcnM --arg v "3.1.0" --arg url "$BOOTSTRAP_URL" '{ignition:{version:$v,config:{replace:{source:$url}}}}' | base64 | tr -d '\n')
+
+az deployment group create --verbose -g "$RESOURCE_GROUP" \
+  --template-file "04_bootstrap.json" \
+  --parameters bootstrapIgnition="$BOOTSTRAP_IGNITION" \
+  --parameters sshKeyData="$SSH_KEY" \
```

Comment:
Is this the same as what we discussed in standup today? That is, the SSH_KEY should be passed through Ignition and not the templates?

Reply:
Yes, but I would prefer to fix that in a followup PR/bz.